tests/testAutopas/testingHelpers/GridGenerator.h

```diff
@@ -48,6 +48,37 @@ class GridGenerator {
                                 const Particle &defaultParticle = autopas::Particle(),
                                 const std::array<double, 3> &spacing = std::array<double, 3>{1, 1, 1},
                                 const std::array<double, 3> &offset = std::array<double, 3>{.5, .5, .5});
+  /**Fills Autopas Object with a Grid of Particles starting at
+   * @param autopas
+   * @param startingPositions
+   * @param particlesPerDim
+   * @param defaultParticle
+   * @param spacing
+   * @param offset
+   *
+   * */
+  template <class Particle, class ParticleCell>
+  static void fillWithParticlesOnR(autopas::AutoPas<Particle, ParticleCell> &autopas,
```

Comment:
what's the point of this function? To place a grid at an arbitrary point in space? This can be achieved by using offset (=offset from 0/0/0) in the existing function.